src/main/java/frc/robot/subsystems/shooterSubsystem/ShooterCommandFactory.java

```diff
@@ -13,8 +13,8 @@
 public class ShooterCommandFactory {
   private final ShooterSubsystem subsystem;
   private ShuffleboardTab shooterTab = Shuffleboard.getTab("ShooterSubsystem");
-  private GenericEntry targetRPM = shooterTab.add("Target RPM", 4000).getEntry();
-  private GenericEntry rightTargetRPM = shooterTab.add("right RPM", 4000).getEntry();
+  private GenericEntry targetRPM = shooterTab.add("Target RPM", 2350).getEntry();
+  private GenericEntry rightTargetRPM = shooterTab.add("right RPM", 1950).getEntry();
```

Comment:
Maybe pull these from speakerConfigList?

Reply:
right now is configured for solely bumper shots, can be adjusted when more turret functionality is available.

Reply:
I meant so you don't have to remember to replace the target RPM numbers in multiple places if this gets retuned. But fine for this PR